docs/examples/theme/GetTheme/README.md

```diff
@@ -0,0 +1 @@
+A component that provides the colors and styling made accessible through the `Theme` component.
```

Comment:
Perhaps worth explicitly mentioning that this is a component that provides the colors/styling by pulling the relevant data out of context, as provided by `Theme`?

Reply:
yea, i think that's probably right. looking at what u have again, i think it's solid.